libraries/src/Component/ComponentHelper.php

```diff
@@ -252,37 +256,37 @@ public static function filterText($text)
 					$filter->blockedAttributes = $customListAttributes;
 				}
 			}
-			// Blacklists take second precedence.
-			elseif ($blackList)
+			// DisallowList take second precedence.
```

Comment:
```suggestion
			// Disallow List take second precedence.
```
Formalise the wording to show as separate words.